src/components/pager/js/Pager.js

```diff
@@ -295,8 +295,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     });
 
     fluid.pager.previousNext.update = function (that, disabledStyle, newModel) {
-        that.previous.toggleClass(disabledStyle, newModel.pageIndex === 0);
-        that.next.toggleClass(disabledStyle, newModel.pageIndex === newModel.pageCount - 1);
+        var previousToggle = (newModel.pageIndex === 0);
+        var nextToggle = (newModel.pageIndex === newModel.pageCount - 1);
+        that.previous.toggleClass(disabledStyle, previousToggle);
+        that.next.toggleClass(disabledStyle, nextToggle);
+        that.previous.children('a').attr('aria-disabled', previousToggle);
+        that.next.children('a').attr('aria-disabled', nextToggle);
```

Comment:
I was reading through some spec documentation for [aria-disabled](https://www.w3.org/TR/wai-aria/states_and_properties#aria-disabled). It says 

> The state of being disabled applies to the current element and all focusable descendant elements of the element on which the aria-disabled attribute is applied.

That would lead me to believe that the `aria-disabled` attribute can be set directly on the previous and next elements.